src/snowflake/cli/plugins/nativeapp/commands.py

```diff
@@ -382,15 +382,25 @@ def app_validate(**options):
 @app.command("events", hidden=True, requires_connection=True)
 @with_project_definition()
 @nativeapp_definition_v2_to_v1
-def app_events(**options):
+def app_events(
+    since: str = typer.Option(
+        default="",
```

Comment:
nit: what happens when we default to ""? Is it from the oldest events in the table? That could be a lot of data, would it make sense to add some kind of numerical limit on the number of events being returned, and default it to unlimited? So `events --since "1 year" --limit 500` would prevent the user from getting too many events displayed before they can kill the command.

Reply:
yup, `--limit` is coming in https://github.com/snowflakedb/snowflake-cli/pull/1370